server/users.js

```diff
@@ -12,9 +12,6 @@ const addUser = ({ id, name, room, isHost }) => {
   if(!name || !room) return { error: 'Username and room are required.' };
   if(existingUser) return { error: 'Username is taken.' };
 
```

Comment:
At some point, we should probably have checks for all the fields that are being passed in